docs/manual/scala/guide/cluster/Projections.md

```diff
@@ -0,0 +1,32 @@
+
+```
+lagom.projection {
+  # when enabled, read-side processors and topic producers start at bootstrap
+  # when disabled, read-side processors and topic producers are added to projection registry but won't start
+  # until a user explicitly sends a request to start it.
+  # TODO: add link to Projection API
+  auto-start.enabled = true
+}
+```
+
+## Projection status
+
+Once up and running, you may query the state of your projections to ensure all the instances are up and running. You can also request a particular projection to stop or even a particular instance (aka worker) in a projection to stop. 
+
+Stopping and starting all the workers or a single worker of a projection is an asynchronous operation. Imagine we had a `ReadSideProcessor` called  `users-by-name` that is reading the journal of a `Users` persistent entity and storing data into a SQL table. To stop that projection we would request the status of all workers to become `Stopped`. This request will propagate across the cluster and each node will make sure that any worker running locally that is part of the `users-by-name` projections is stopped. As workers switch from `Started` to `Stopped` they report back and we can `observe` their new status.
+
+To request and modify the status of your projections simply use the `projections` instance provided by any of the [[`Components`|ScalaComponents]] that create projections (any persistence implementation or broker API implementation).
+
+The _requested status_ is a volatile, in-memory value but it is replicated across your cluster. Only when the whole cluster is restarted you may have to request a particular status again. Also, because of the replicated nature, the _requested status_ may be overwriten by multiple nodes on your cluster at the same time. The implementation is such that [last writer wins](https://doc.akka.io/docs/akka/current/distributed-data.html#data-types).
+
+When a new instance of a projection worker is created the rules to decide its status are:
+
+* if there's worker requested status, use it; if not
+* use the default value in `application.conf`
+
+Because the _requested status_ is a distributed, in-memory value, there is an edge case you will have to consider when you need a worker to be stopped. When starting a new node and having that node join a n existing cluster, it is possible that some workers are spawned in the node before the _requested status_ value is received from the peer nodes. In that case, the default `lagom.projection.auto-start.enabled` will be used to decide if the spanwed worker should be stopped or started. If your default is `enabled = true` but the in-memory, replicated value is `Stopped` then there's a race condition and you could observe your worker _start-and-then-stop_. To prevent the  _start-and-then-stop_ behavior, opt out of `lagom.projection.auto-start.enabled = true` and always handle worker startup using the methods `startAllWorkers`/`startWorker` on the `Projections`AP.
```

Comment:
```suggestion
Because the _requested status_ is a distributed, in-memory value, there is an edge case you will have to consider when you need a worker to be stopped. When starting a new node and having that node join an existing cluster, it is possible that some workers are spawned in the node before the _requested status_ value is received from the peer nodes. In that case, the default `lagom.projection.auto-start.enabled` will be used to decide if the spawned worker should be stopped or started. If your default is `enabled = true` but the in-memory, replicated value is `Stopped` then there's a race condition and you could observe your worker _start-and-then-stop_. To prevent the  _start-and-then-stop_ behavior, opt-out of `lagom.projection.auto-start.enabled = true` and always handle worker startup using the methods `startAllWorkers`/`startWorker` on the `Projections` API.
```